tests/integration/synapseclient/core/test_download.py

```diff
@@ -191,12 +191,11 @@ async def test_download_cached_file_to_new_directory(
 
             # WHEN I download the file to another location
             updated_location = os.path.join(
-                os.path.dirname(original_file_path), "subdirectory"
+                os.path.dirname(original_file_path),
+                "subdirectory/asd/asdas/dasd/asdas/dasdasd",
```

Comment:
I remember you had written this as part of your demo today and it should be reverted back to `"subdirectory"` haha